src/components/Shared/Filters/FilterPanel.tsx

```diff
@@ -412,45 +416,17 @@ export const FilterPanel: React.FC<FilterPanelProps & BoxProps> = ({
                 return {
                   ...acc,
                   [key]: value.split(',').map((enumValue) => {
-                    switch (enumValue) {
-                      // Status
-                      case 'active':
-                        return ContactFilterStatusEnum.Active;
-                      case 'hidden':
-                        return ContactFilterStatusEnum.Hidden;
-                      case 'null':
-                        return ContactFilterStatusEnum.Null;
-                      case 'Appointment Scheduled':
-                        return ContactFilterStatusEnum.AppointmentScheduled;
-                      case 'Ask in Future':
-                        return ContactFilterStatusEnum.AskInFuture;
-                      case 'Call for Decision':
-                        return ContactFilterStatusEnum.CallForDecision;
-                      case 'Contact for Appointment':
-                        return ContactFilterStatusEnum.ContactForAppointment;
-                      case 'Cultivate Relationship':
-                        return ContactFilterStatusEnum.CultivateRelationship;
-                      case 'Expired Referral':
-                        return ContactFilterStatusEnum.ExpiredReferral;
-                      case 'Never Ask':
-                        return ContactFilterStatusEnum.NeverAsk;
-                      case 'Never Contacted':
-                        return ContactFilterStatusEnum.NeverContacted;
-                      case 'Not Interested':
-                        return ContactFilterStatusEnum.NotInterested;
-                      case 'Partner - Financial':
-                        return ContactFilterStatusEnum.PartnerFinancial;
-                      case 'Partner - Pray':
-                        return ContactFilterStatusEnum.PartnerPray;
-                      case 'Partner - Special':
-                        return ContactFilterStatusEnum.PartnerSpecial;
-                      case 'Research Abandoned':
-                        return ContactFilterStatusEnum.ResearchAbandoned;
-                      case 'Unresponsive':
-                        return ContactFilterStatusEnum.Unresponsive;
-                      default:
-                        return ContactFilterStatusEnum.Null;
-                    }
+                    // Status
+                    // Check if saved filter (enumValue) is either the status in all caps (PARTNER_FINANCIAL) or (Partner - Financial)
```

Comment:
@dbenton9 @dr-bizz I made this change in the filters a while ago. The front end should grab the correct saved filters whether the status is (PARTNER_FINANCIAL) or (Partner - Financial). I'm not sure what the API will be providing.

Reply:
Makes perfect sense that you originally set this up with (Partner - Financial), as we're transitioning the db to constant values in snake case format (partner_financial) I would anticipate the FE also adapting to handle this change in the db data.